test/e2e/framework/get-kubemark-resource-usage.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strings"
 )
 
+// KubemarkResourceUsage is structure for handling resource usage of kubemark.
```

Comment:
tiny nit / suggestion:
```suggestion
// KubemarkResourceUsage is a struct for tracking resource usage of kubemark.
```

Reply:
Thanks for your review @BenTheElder :-) I am applying this on all related places.